crates/matrix-sdk-indexeddb/src/crypto_store/mod.rs

```diff
@@ -159,6 +159,53 @@ impl From<IndexeddbCryptoStoreError> for CryptoStoreError {
 
 type Result<A, E = IndexeddbCryptoStoreError> = std::result::Result<A, E>;
 
+/// Defines an operation to perform on the database
+enum DbOperation {
```

Comment:
For consistency with `PendingChanges` suggested below, could this be `PendingOperation`?